lib/sycamore/sycamore/tests/unit/transforms/compare_detr_impls.py

```diff
@@ -1,22 +1,18 @@
 from sycamore.utils.cache import Cache
```

Comment:
This no longer compares impls, right? Should we rename?

Reply:
Done